src/components/Item/Item.js

```diff
@@ -145,6 +146,9 @@ export const Item = ({
 
   const avatar = <img {...{ src: items[id] }} alt={name} />
 
+  const sellPrice =
```

Comment:
This is exactly what I was thinking! I think we're going to need `sellPrice` instead of `adjustedValue` here on like 294 as well: https://github.com/jeremyckahn/farmhand/blob/cea1cb22029552865edf3a3f5c45556e57dd9f20/src/components/Item/Item.js#L288-L300

Once that's fixed, I think this should be all good! 🙂

Reply:
oh wow i did not see that one at all!

Reply:
actually, i'm not sure this one was supposed to change 🤔 . this is for purchasing, right? we only want to affect sell price I think

Reply:
Ah! Great catch, you're right. Thanks for scrutinizing the code extra closely here! 😅